### PR TITLE
Edit Multisig Page

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@Tbaut let me know what you think, i added Multix in the multisig wiki page

File: docs/learn/learn-account-multisig.md [hunk 1]
```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
...
+
+:::info Walk-Through Video Tutorial
+
+See [this video tutorial](https://www.youtube.com/watch?v=APxPsawebJw) and
+[this article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d)
+for more information about using [Multix](https://multix.chainsafe.io/) for multisig accounts.
+
+:::
+
+> Note that Multix is currently only available on Rococo (Polkadot's Testnet). Soon it will be
```
Comment (Tbaut) [inline docs/learn/learn-account-multisig.md]: Thanks for the shoutout. You can remove this, it will be outdated in the next days anyway.

**Review by Tbaut — APPROVED** [review]

**Comment by Tbaut** [comment]
Thank you so much for the mention and very positive review :pray:

File: docs/learn/learn-account-multisig.md [hunk 2]
```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
...
+> available on Westend, Kusama and Polkadot.
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]: Addressing review comments

```suggestion
```

File: docs/learn/learn-account-multisig.md [hunk 3]
```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
...
+
+The [Multix tool](https://multix.chainsafe.io/) is an easy-to-use interface to manage complex
+multisigs. The tool is part of
+[the open-source Polkadot/Kusama Tech Stack](https://wiki.polkadot.network/docs/build-open-source#user-interface).
+Besides being user-friendly, Multix provides several benefits compared to the Polkadot-JS UI:
+
+- When one person creates the multisig all the signatories see the multisig in their interface.
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]:
```suggestion
- When one person creates the multisig account, all the signatories see the multisig account on their interface.
```

File: docs/learn/learn-account-multisig.md [hunk 4]
```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
...
+- To confirm a transaction there is no need to pass around the call data. Such information is
+  retrieved from the chain and displayed to the users.
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]:
```suggestion
- There is no need to pass around the call data to confirm a transaction. Such information is
  retrieved from the chain and displayed to the users.
```

File: docs/learn/learn-account-multisig.md [hunk 5]
```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
...
+- Thanks to the power of [pure proxies](../learn/learn-proxies.md#anonymous-proxy-pure-proxy),
+  adding and removing signatories or adjusting the threshold is now possible without creating new
+  multisigs or understand how pure proxies work.
```
Comment (profradha) [inline docs/learn/learn-account-multisig.md]:
```suggestion
- Thanks to the power of [pure proxies](../learn/learn-proxies.md#anonymous-proxy-pure-proxy),
  adding and removing signatories or adjusting the threshold is now possible without creating new
  multisigs and Multix provides an intuitive interface to work with pure proxies.
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**